src/main/java/com/minecolonies/network/messages/BlockParticleEffectMessage.java

```diff
@@ -76,11 +67,13 @@ public IMessage onMessage(BlockParticleEffectMessage message, MessageContext ctx
     {
         if(message.side == BREAK_BLOCK)
         {
-            Minecraft.getMinecraft().effectRenderer.addBlockDestroyEffects(new BlockPos(message.x, message.y, message.z), message.block.getDefaultState()); //todo check default state, mw, trans 1.7
+            //TODO check default state, mw, trans 1.7
+            Minecraft.getMinecraft().effectRenderer.addBlockDestroyEffects(message.pos, message.block.getDefaultState());
         }
         else
         {
-        	FMLClientHandler.instance().getClient().effectRenderer.addBlockHitEffects(new BlockPos(x, y, z), EnumFacing.getFront(side)); // TODO: test if this works
+            // TODO: test if this works
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Comment has incorrect indentation level 12, expected is 9, indentation should be the same level as line 76. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.indentation.CommentsIndentationCheck)